internal/generate/olm-catalog/descriptor/parse_test.go

```diff
@@ -18,136 +18,65 @@ import (
 	"reflect"
 	"testing"
 
-	"github.com/operator-framework/operator-sdk/internal/annotations"
-	"k8s.io/gengo/types"
-
-	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	operatorsv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
 )
 
 func TestParseResource(t *testing.T) {
 	cases := []struct {
 		description string
-		input       string
```

Comment:
Many tests were removed here. If it is just a refractory, should we not keep doing the sea tests/checks? 
Can we ensure that the same check still made?

Reply:
The tests I removed were testing things that the markers system does for us, so they are not relevant any more.